test/helpers.js

```diff
@@ -53,17 +47,14 @@ export function tryToReadFile(file) {
 	}
 }
 
+const { window } = new JSDOM('<main></main>');
+global.document = window.document;
```

Comment:
It has been a [source](https://github.com/tmpvar/jsdom/issues/1825#issuecomment-311195478) of non-security [issues](https://github.com/tmpvar/jsdom/issues/1620), but I can understand if it feels unnecessary to change now.